src/routes/ExercisesScreens.tsx

```diff
@@ -47,6 +67,17 @@ const ExercisesScreen = ({ route, navigation }: ExercisesScreenProps): JSX.Eleme
 
   return (
     <Root>
+      <ConfirmationModal
+        visible={isModalVisible}
+        setVisible={setIsModalVisible}
+        text="Ups, this exercise is still unlocked."
```

Comment:
As the app is in german, we use the `src/constants/labels.json`. Please create entries there, which you use here. It is fine to put the english wording in the labels.json. 
Please put all text that is visible to the user into the labels.json-file.
I will later adjust them to german.